lib/server/forwardClientRequestToApi.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+var request = require("request-promise");
+
+function forwardClientRequestToApi(apiHost){
+    return function(req,res){
+        var options = {
+            uri: apiHost + req.url.replace(/^\/api/,''),
```

Comment:
should we use req.originalUrl (http://expressjs.com/4x/api.html#req.originalUrl) instead of url?